components/manager/Dockerfile

```
@@ -35,12 +35,15 @@ RUN CGO_ENABLED=0 GOOS=linux go build -tags netgo \
   -o /manager ./components/manager/cmd/app/main.go
 
 # Stage 2: Final image using minimal runtime dependencies
-FROM alpine:3.24
+FROM alpine:3.24.1
 
 WORKDIR /app
 
-# Install certificates/timezone data and curl for health checks
-RUN apk add --no-cache zlib ca-certificates tzdata curl && \
+# Install certificates/timezone data, patched OpenSSL runtime, and curl for health checks
+RUN apk add --no-cache \
```

Hadolint (2.15.1)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)